<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorDecode</class>
 <widget class="QDialog" name="ErrorDecode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="PushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Shoooot!</string>
   </property>
  </widget>
  <widget class="TextBrowser" name="textBrowser_output">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="LineEdit" name="lineEdit_input_num">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="TextEdit" name="textEdit_data_struct">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="TableWidget" name="widget_table" native="true">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>311</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
